docs/includes/net-standard-table.md

```diff
@@ -1,10 +1,19 @@
 | Platform Name              | Alias       |        |        |        |        |        |        |        |       |
 |----------------------------|-------------|--------|--------|--------|--------|--------|--------|--------|-------|
-| .NET Standard              | netstandard | 1.0    | 1.1    | 1.2    | 1.3    | 1.4    | 1.5    | 1.6    | 2.0   |
+| .NET Standard              | netstandard | [1.0]  | [1.1]  | [1.2]  | [1.3]  | [1.4]  | [1.5]  | [1.6]  | [2.0] |
```

Comment:
Should the .NET Standard row effectively become the header? This would underline stronger that this table really shows the relationship between all platforms and .NET Standard. Otherwise one can read the table in such a way that all platforms are somehow related to each other -- which isn't the case. In other words, .NET Standard isn't a peer of the other rows; it is the driver for the column.